challenge5/index.html

```diff
@@ -0,0 +1,292 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="https://fonts.googleapis.com/css2?family=Mulish:wght@400;500&display=swap" rel="stylesheet">
+  <script src="https://cdn.tailwindcss.com"></script>
+  <title>Document</title>
+  <script>
+    tailwind.config = {
+      theme: {
+        extend: {
+          colors: {
+            primary_color: "#FFFFFF",
+            secondary_color: "#458FF6",
+            top_gradient: "#5A98F2",
+            bottom_gradient: "#67C3F3",
+            background: "#FAF9F6",
+          },
+          lineHeight: {
+            12: "60px",
+          },
+          rotate: {
+            160: "160deg",
+          },
+          padding: {
+            11.5: "46.5px",
+            15: "60px",
+          },
+          fontFamily: {
+            sans: ['Mulish', 'sans-serif'],
+          },
+          width: {
+            65: "259px",
+            button: "199px",
+          },
+          fontSize: {
+            button: ["18px", "60px"],
+          },
+        },
+      },
+    };
+  </script>
+</head>
+
+<body class="bg-background z-0 font-sans">
+  <header class="flex flex-row px-8 pt-2 pb-15 justify-between">
+    <div class="flex flex-row">
+      <img src="./assets/icons/logo.svg" alt="Logo" />
+    </div>
+    <div>
+      <div class="hidden md:flex">
+        <ul class="flex flex-row gap-x-5 pl-10">
+          <li class="font-medium">Home</li>
+          <li class="text-gray-400 font-medium">Find a doctor</li>
+          <li class="text-gray-400 font-medium">Apps</li>
+          <li class="text-gray-400 font-medium">Testimonials</li>
+          <li class="text-gray-400 font-medium">About us</li>
+        </ul>
+      </div>
+      <div class="flex md:hidden">
+        <img src="./assets/icons/List.svg" alt="Menu" />
+      </div>
+    </div>
+  </header>
+  <main class="flex flex-col overflow-hidden">
+    <section class="relative flex flex-col md:flex-row pl-9 pr-11.5 justify-center grow">
+      <div class="flex flex-col pr-10 pb-9 gap-y-6 self-center max-w-md">
+        <h1 class="pr-12 text-3xl font-bold">
+          Virtual healthcare for you
+        </h1>
+        <p class="text-lg font-light text-gray-400">
+          Trafalgar provides progressive, and affordable healthcare,
+          accessible on mobile and online for everyone
+        </p>
+        <button class="bg-secondary_color text-white rounded-full text-button px-10 w-button hover:opacity-80">
+          Consult today
+        </button>
+      </div>
+      <div class="flex flex-auto md:w-2/3 max-w-xl">
+        <img src="./assets/imgs/imgSection1.svg" alt="Illustration" class="flex flex-auto md:w-2/3" />
```

Comment:
For decorative images we should use `alt=""` since it is not relevant for screen readers.